build.rs

```diff
@@ -18,9 +18,6 @@ fn main() {
   // variable can lead to broken links when switching rusty_v8
   // versions.
   let envs = vec![
-    "CARGO",
-    "CARGO_MANIFEST_DIR",
-    "CARGO_PKG_VERSION",
```

Comment:
This one is actually necessary to ensure rebuilds on version changes.

Reply:
Actually, I guess cargo will already rebuild the buiild script on version changes, so disregard that.